api/src/wfl/service/firecloud.clj

```diff
@@ -15,7 +15,7 @@
 (def ^:private workspace-api-url
   (partial firecloud-url "api/workspaces"))
 
-(defn ^:private get-workspace-json [& parts]
```

Comment:
I'd prefer if you didn't expose this and instead write a function called `get-workspace` that had the precondition that its argument (the workspace) was `some?`

Reply:
Ok - it originally was that way but I was asked to just expose the private method.

Reply:
Reverted